include/pybind11/detail/common.h

```diff
@@ -211,8 +211,9 @@
 #endif
 
 #include <Python.h>
-#if PY_VERSION_HEX < 0x030500f0
-#    error "PYTHON 2 IS NO LONGER SUPPORTED. pybind11 v2.9 was the last to support Python 2."
+#if PY_VERSION_HEX < 0x03060000
+#    error                                                                                        \
+        "PYTHON < 3.6 IS NO LONGER SUPPORTED. pybind11 v2.9 was the last to support Python 2 and 3.5."
```

Comment:
Ouch, that's weird. Probably should shorten the line a little. UNSUPPORTED for example.